src/pd_initialize.jl

```diff
@@ -22,14 +26,14 @@ function pd_initial(NA, lambda, imsz)
     return H, rho, theta
 end
 
-function zern_initial(img, H, rho, initial_param)
-    n, _, lambda, _, _ = initial_param
-    z = size(img)[3]
+function zern_initial(H, rho, initial_param, imsz)
+    (; n, NA, lambda, Z_orders) = initial_param
+    z = imsz[3]
     zFrame = (1-Int(z / 2)):Int(z / 2)
```

Comment:
```suggestion
    iseven(z) || throw(ArgumentError("imsz[3] must be an even integer"))
    zFrame = (1-Int(z / 2)):Int(z / 2)
```

If it's not even, the next line will fail with an `InexactError`, and I worry that will be pretty opaque: does this represent a bug in the package or user error? That's an especially nasty problem if we don't document anywhere that only even sizes are supported.

Reply:
I figure a way to fix this.
`zFrame = (1:z) .- Int(floor(z/2))`
Should work as expected and we don't need to worry about not been even number.